index.html

```diff
@@ -1499,7 +1496,7 @@ <h3>Credential Subject</h3>
           <dd>
 The value of the <code>credentialSubject</code> <a>property</a> is defined as
 a set of objects that contain one or more properties that are each related to a
-<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
+<a>subject</a> of the (verifiable) <a>credential</a>. Each object MAY contain an
```

Comment:
```suggestion
<a>subject</a> of the <a>credential</a> or <a>verifiable credential</a>. Each
object MAY contain an
```

Reply:
Ok accepted. But please note that the current recommendation is inconsistent throughout the entire text in that sometimes it refers to credential and sometimes to verifiable credential, almost randomly it would seem.

Reply:
I guess we have some careful review to make, to ensure that all of these correctly refer to one or the other or both, possibly with additional text to explain why any given instance is one or the other or both.

Reply:
Agreed. But its not a simple quick task. Because the original text was crafted over several years by different authors, with multiple edits of different sections as well, then many inconsistencies have crept in. I used the shorthand (verifiable)credential to imply that both credential and verifiable credential are equally applicable in this sentence.

Reply:
I understand the shorthand, but many readers may not, and shorthand should generally be avoided in technical docs such as this, unless the intended meaning of that shorthand is clearly defined *and* every instance of the shorthand links back to that definition.

Reply:
Perhaps introducing this shorthand into the definitions section might be advantageous. It would make subsequent editing simpler, and it would resolve the "and" or "or" grammatical issue below :-)

Reply:
The immediate concern I have with that is making sure the definitions tools can handle the parenthetical `<a>(verifiable) credential</a>` differently than `<a>verifiable credential</a>`. I don't know whether they would recognize that difference today.

The definitely available way to handle this is the way I've already been aiming toward -- i.e., `<a>verifiable credential</a> and/or <a>credential</a>` or `<a>credential</a> and/or <a>verifiable credential</a>`.